desktop/appcontainer/VersionBar.ts

```diff
@@ -55,7 +55,7 @@ export const versionBar = hoistCmp.factory({
                 Icon.wrench({
                     omit: isAdminApp || !XH.getUser().isHoistAdminReader,
                     title: 'Open Admin Console',
-                    onClick: () => window.open('/admin')
+                    onClick: () => XH.openInTab('/admin', 'xhAdmin')
```

Comment:
What does passing `xhAdmin` mean in this context?

Reply:
Please disregard, I caught up on the Slack channel about how this relates to session storage and tabId 👍